main.py

```diff
@@ -10,8 +12,46 @@
 from config import bot_token
 
 
-class AnyContentType:
-    def __contains__(self, item): return True
+# Callback data dict keys are converted to UPPERCASE abbreviations; values are converted to lowercase
+callback_data_contractions = {'type': 'T', 'operator_ids': 'OIS', 'conversation_end_moment': 'CEM', 'mood': 'M',
+                              'conversation_rate': 'cr', 'better': 'b', 'same': 's', 'worse': 'w'}
+
+
+def contract_callback_data(d, converter=None):
+    if converter is None:
+        converter = callback_data_contractions
+
+    e = {}
+    for key, value_ in d.items():
+        try:
+            value = converter.get(value_, value_)
+        except TypeError:
+            value = value_
+
+        e[converter.get(key, key)] = value
+    return e
+
+def contract_callback_data_and_jdump(d, converter=None):
+    return json.dumps(contract_callback_data(d, converter), separators=(',', ':'))
+
+def decontract_callback_data(d, converter=None):
+    if converter is None:
+        # Use inverted `callback_data_contractions` by default
+        converter = {v: k for k, v in callback_data_contractions.items()}
+    return contract_callback_data(d, converter)
+
+def jload_and_decontract_callback_data(d, converter=None):
+    return decontract_callback_data(json.loads(d), converter)
+
+
+# Used to reduce number of digits in the `total_seconds` sent as a callback
+local_epoch = datetime(2020, 11, 1)
+
+def secs_since_local_epoch(dt):
+    return int((dt - local_epoch).total_seconds())
+
+def dt_from_local_epoch_secs(secs):
+    return local_epoch + timedelta(seconds=secs)
```

Comment:
Should be renamed to `seconds_since_local_epoch` and `datetime_from_local_epoch_secs` respectively